config/src/test/java/org/springframework/security/config/annotation/web/configurers/SessionManagementConfigurerServlet31Tests.java

```diff
@@ -100,15 +89,17 @@ public void changeSessionIdDefaultsInServlet31Plus() throws Exception {
 		CsrfToken token = repository.generateToken(request);
 		repository.saveToken(token, request, response);
 		request.setParameter(token.getParameterName(), token.getToken());
-		when(ReflectionUtils.findMethod(HttpServletRequest.class, "changeSessionId"))
-				.thenReturn(method);
 
 		loadConfig(SessionManagementDefaultSessionFixationServlet31Config.class);
 
 		springSecurityFilterChain.doFilter(request, response, chain);
 
-		verifyStatic(ReflectionUtils.class);
-		ReflectionUtils.invokeMethod(same(method), any(HttpServletRequest.class));
+
+		Assert.assertNotEquals(id, request.getSession().getId());
```

Comment:
Would you do me a favor and please use AssertJ instead? `assertThat(request.getSession().getId()).isEqualTo(id)`

Reply:
done